Getting-Started.md

```diff
@@ -709,4 +853,58 @@ Kibiter then automatically generates a query as shown below.
 
 > :warning: **If you change the structure or delete it** you won't see the menu.
 
-> **Note**: the `<id_of_the_dashboard>` must be the identifier that Kibana/Kibiter put to a saved dashboard.
\ No newline at end of file
+> **Note**: the `<id_of_the_dashboard>` must be the identifier that Kibana/Kibiter put to a saved dashboard.
+
+#### Dump a mapping/data from an index [&uarr;](#dump-data/mapping-)
+
+**elasticdump installation**
+
+```
+sudo apt install npm
+sudo -i
+npm install elasticdump -g
+
+```
+
+![elasticdump_installation](https://user-images.githubusercontent.com/53489999/77806917-23786500-70ac-11ea-9d09-dbdb38caf6aa.png)
```

Comment:
This figure can be removed since the command above already provides the user to install elasticdump.

Reply:
Okay..Will remove it. Thanks!